doc/changes/changes_0.1.0.md

```diff
@@ -0,0 +1,213 @@
+# r-exasol 0.1.0, released t.b.d.
+
+Code name: t.b.d.
+
+## Summary 
+
+t.b.d.
+
+## Features / Enhancements
+
+
+* 1st version
```

Comment:
Please check the format (https://github.com/exasol/release-droid/blob/main/doc/user_guide/templates/changes_file_template.md). 
It's `# ISSUE_NR: Descriptioin what was changed`.

This file is for users. So it should describe what was changed on a higher level.
For example: `#123: Added bulk loading`.
So it's one entry per issue, not per push.

Reply:
Yes, the history is quite a problem. In the early stages of this project they didn't comment the tickets in the commit messages. So I see 2 options:

- Add some kind of generic message like: Initial implementation
- Keep it as is

Reply:
I think `Initial implementation` is fine.